sdk/security/keyvault/azadmin/backup/models.go

```diff
@@ -70,12 +70,12 @@ type SASTokenParameters struct {
 	StorageResourceURI *string
 
 	// The SAS token pointing to an Azure Blob storage container
-	Token *string `json:"token,omitempty"`
```

Comment:
Strange this didn't get removed in the first PR

Reply:
When I rebased my changes, I didn't regenerate the code, I just accepted what was in HEAD which was built with the older code generator.

Reply:
Makes sense!